examples/nats-streaming-queue-group/subscriber.out

```diff
@@ -0,0 +1,10 @@
+# To run this sample, navigate to the directory that contains the
+# `.bal` file, and execute the `ballerina run` command.
+# `queue-group.bal` contains three services belonging to the same
+# queue group.
+# When several messages are published, it can be noticed that
+# each message is received by only one queue group member.
+$ ballerina run queue-group.bal
+Message Received to third queue group member: First Message
+Message Received to second queue group member: Second Message
```

Comment:
```suggestion
Message received by the second member of the queue group: Second Message
```

Reply:
`queue group` is a well known term for NATS. I don't think we should rephrase this way.